plugin/types.go

```diff
@@ -15,7 +15,15 @@ or referenced via the API - and tools for controlling how its data is cached.
 Implementing the Parent interface displays that resource as a directory on the filesystem.
 Anything that does not implement Parent will be displayed as a file.
 
-The Readable interface gives a file its contents when read via the filesystem.
+The Readable interface allows reading data from an entry via the filesystem. The Writable
+interface allows sending data to the entry.
+
+Wash distinguishes between two different patterns for things you can read and write. It considers
+a "file-like" entry to be one with a defined size (so the `size` attribute is set when listing the
```

Comment:
OK, I agree about using `size` alone for file-like things. But I think we should remove `size` from the `metadata/data.json` files because those aren't really "files" (editing them makes no sense) so they shouldn't be treated as such.

Reply:
I think editing metadata does make sense for some things: the metadata is a reflection of a mix of state and configuration, and you may be able to make changes to that configuration. Editing those values to change the configuration would make sense. This is especially true for everything in Kubernetes.

Reply:
Right, but what you write isn't going to strictly match what's read -- there isn't a 1:1 symmetry between the content because we output pretty-printed JSON. But I don't have strong feelings about keeping size around either so up to you.

Reply:
Ok, that makes sense.